packages/compass-query-bar/src/components/query-bar/query-bar.tsx

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import {
+  Button,
+  Icon,
+  Label,
+  MoreOptionsToggle,
+  css,
+  cx,
+  focusRingStyles,
+  focusRingVisibleStyles,
+  spacing,
+  uiColors,
+} from '@mongodb-js/compass-components';
+
+const queryBarStyles = css({
+  display: 'flex',
+  alignItems: 'center',
+  gap: spacing[2],
+  padding: spacing[2],
+  border: `1px solid ${uiColors.gray.light2}`,
+  borderRadius: '6px',
```

Comment:
Is this radius shared with other components, for example something in the Aggregation Tab?

Reply:
Yes, how about an exported `toolbarBorderRadius` variable in the Toolbar component in compass-components so we can share it? I'll merge this and can include in next pr if that sounds like a good move, open to other ways to share too. Probably not worth it's own component I think, but the `border` is also duplicated.